setup.sh

```diff
@@ -39,100 +39,61 @@ done
 
 set -u # error on use of undefined variable
 
-if ! command -v brew > /dev/null ; then
+if ! command -v gfortran > /dev/null ; then
+  echo "This script assumes usage of gfortran to compile and build"
+  echo "When using other Fortran compilers, please ensure you have fpm downloaded and in your path"
+  echo "Then use the following command to install inference-engine: fpm build --flag \"-fcoarray=single\"."
+  echo "Please replace the coarray flag with appropriate coarray flag for your compiler"
```

Comment:
```suggestion
  echo "Please replace the text from the opening angular bracket to the ending bracket  with appropriate flag, if any, that your compiler requires to support Fortran's parallel features (e.g., -fcoarray=single for single-image builds with gfortran)."
```

Reply:
This if clause was unintentionally left in. Based on your reminder yesterday that `fpm` automatically provides the correct coarray flag for each Fortran compiler, this clause is not necessary and I have removed it.